website/src/pages/versions.js

```diff
@@ -20,8 +20,9 @@ function Version() {
   const pastVersions = versions.filter(
     (version) => version !== latestVersion && version.name !== 'current',
   );
-
+  const stableVersion = pastVersions.shift();
```

Comment:
```suggestion
  const stableVersion = pastVersions.shift() ?? latestVersion;
```

Reply:
(just an idea, this wn't lead to a good deploy preview versions page for the bootstrap theme, but probably not a big deal anyway)

Reply:
This solution is ok? https://github.com/facebook/docusaurus/pull/3577/commits/84aa3c7d463b6030e338441978f4ff67ee28f140

Reply:
looks even better ;)